src/nl/surf/eduhub_rio_mapper/api.clj

```diff
@@ -214,14 +232,26 @@
         (wrap-json-response)
         (wrap-logging)
         (wrap-trace-context)
-        (defaults/wrap-defaults defaults/api-defaults))))
+        (defaults/wrap-defaults defaults/api-defaults)
+        (wrap-server-status)
+        (wrap-request-counter))))
```

Comment:
deze laatste twee moeten denk ik andersom -- als je het zo laat en er komen veel concurrent requests stopt het proces nooit. misschien is het handiger om ze samen te voegen?